docs/userguide/icu/editing.md

```diff
@@ -159,6 +159,7 @@ To activate the version of Ruby:
 ```bash
 rbenv init  # OR:  eval "$(rbenv init -)"
 rbenv shell <version-num>
+rbenv versions  # verify the specified version is in use
```

Comment:
Comment about a different file:
I see you moving the glossary into the icu/ folder, but in the navbar it's still at top level. They should be consistent; not sure which is better. I am ok with moving it in the navbar under ICU.